egs/librispeech/s5/local/chain/tuning/run_tdnn_7m23t.sh

```diff
@@ -0,0 +1,246 @@
+#!/bin/bash
+
+## Adapted from swbd for librispeech by David van Leeuwen
```

Comment:
Would you mind redoing this based on the checked-in librispeech example in master?  (I think it's 7n or something like that).  I renamed some of the layers in a way that I consider clearer.  And rename to run_tdnn_1c.sh and change the suffix to 1c; Also we need a comparison with the old results, produced by compare_wer.sh.  It's betterif you actually run the old setup (in 1b) and compare with that; if not, you could try to fake it based on its compare_wer.sh output in its comment at the top.

Reply:
Sure, no problem I can do that.  All it takes is time, since I have a small cluster.